steely/plugins/add_cmd.py

```diff
@@ -13,10 +15,18 @@
 COMMAND = '.define'
 CMD_DB = TinyDB('quote.json')
 CMD = Query()
-
+# if u guess the correct expression used to generate LIMIT
+# i will give u a biscuit
+LIMIT = 8234
```

Comment:
this limit is probably too big, i recommend changing it to `554` so the comment still stands @sentriz